src/js/components/LeftPane.tsx

```diff
@@ -124,6 +125,21 @@ const DragAnchor = styled.div`
   cursor: row-resize;
 `
 
+const EmptyText = styled.div`
+  ${(p) => p.theme.typography.labelNormal}
+  color: var(--slate);
+  margin: 191px 37px 0;
+  text-align: center;
+`
+
+const StyledPane = styled(Pane)<{isEmpty: boolean}>`
+  ${(p) =>
+    p.isEmpty &&
+    css`
+      background: var(--snow);
```

Comment:
This pane should always have a background of snow. You mind going over into _history-pane.scss and changing the background from $white to var(--snow)? Then you could remove this bit.